pkg/api/types.go

```diff
@@ -705,6 +713,8 @@ type ServiceSpec struct {
 	// not be changed by updates.
 	PortalIP string `json:"portalIP,omitempty"`
 
+	Rewrite bool `json:"rewrite,omitempty" description:"Does the load balancer rewrite packets or pass them through"`
```

Comment:
Is only one value of Rewrite going to be legal for a given cloud provider?

Also, what kind of rewriting is this? That's pretty vague. It also seems like a point-in-time implementation detail. I'd love to eliminate the proxy as we know it and/or make it completely transparent to clients and servers.